docs/photoniq/vwrs/configure-vwrs-edgeworker.md

```diff
@@ -62,85 +62,6 @@ A request configuration is an optional object that contains configuration parame
 
 VWRs can use device fingerprinting techniques to help mitigate cookie sharing and replay attacks. By default, a basic fingerprint is generated using information derived from the device object ([https://techdocs.akamai.com/edgeworkers/docs/device-object](https://techdocs.akamai.com/edgeworkers/docs/device-object)). If stronger device fingerprinting is required, then you can override this logic with your own, or configure it to use one of the fingerprinting techniques available in Akamai’s security products.
 
-## Priority
-
-You can customize a waiting room with up to ten priorities, the priorities being positive numbers from 1 through 255. Configure the priorities using the `POST/PUT/PATCH /api/vwr/v1/domains` REST API.
-
-For instance, consider configuring three priorities with values of 1, 20, and 255. To accomplish this, you can use the `PATCH /api/vwr/v1/domains` REST API to update the domain:
-
-```bash
-curl -X 'PATCH' \
-   https://api-vwr-service.gdn-akamai.com/api/vwr/v1/origins/example3.com \
-  -H 'accept: application/json' \
-  -H 'Content-Type: application/json' \
-  -H 'authorization: test_key.12345678abcdefghijklABCDEFGHIJKL' \
-  -d '{
-  "priority": [1, 20, 255]
-}'
-```
-
-You have now successfully configured the waiting room to support priorities. To prioritize the traffic, you must pass the priority to the `handleVwrsRequest()` and `handleVwrsResponse()` functions. Here's a template demonstrating how to include a priority with the request in the `handleVwrsRequest()` call:
-
-```js
-export async function onClientRequest(request: EW.IngressClientRequest) {
-  await client.handleVwrsRequest(request, { priority: 1 });
-}
-```
-
-This template sets the priority of this request to 1.
-
-Several methods are available to retrieve the actual priority, such as query parameters, headers, or cookies. The precise method is entirely up to the specific implementation needs of the application.
-
-### Query Parameter Example
-
-```bash
-GET /checkout?waiting-room-priority=20
-```
-
-The example adds the request to the waiting room with a priority of 20.
-
-```js
-export async function onClientRequest(request: EW.IngressClientRequest) {
-  const queryParams = new URLSearchParams(request.query);
-  const reqPriority = parseInt(queryParams.get("waiting-room-priority"), 10);
```

Comment:
shouldn't this value be 20?
there are few other places that have 10 in this PR.

Reply:
@manoj-desilva that is redix means parseInt function will return explicitly specify base-10 integer so it is okay.

Reply:
Thanks @naimishvora-macrometa for clarification.